generators/workspace/index.js

```diff
@@ -401,33 +435,45 @@ class Generator extends Base {
   }
 
   writing() {
+
+    const {plugins, dependencies, devDependencies, scripts, watch, partlyRepos} = this._generateWebDependencies(this.props.modules);
+    const sdeps = this._generateServerDependencies(this.props.modules);
+    const dockerWebHint =
+    '  #  web:\n' +
+    '  #    build:\n' +
+    '  #      context: ./deploy/web\n' +
+    '  #      dockerfile: deploy/web/Dockerfile\n' +
+    '  #    ports:\n' +
+    '  #      - \'8090:80\'\n' +
+    '  #    volumes:\n' +
+    '  #      - \'./bundles:/usr/share/nginx/html\'\n' +
+    '  #    depends_on:\n' +
+    '  #      - api\n';
+
     // save config
     this.fs.extendJSON(this.destinationPath('.yo-rc-workspace.json'), {
       modules: this.props.modules,
-      defaultApp: this.props.defaultApp
+      defaultApp: this.props.defaultApp,
+      frontendRepos: plugins,
+      partlyRepos: partlyRepos,
+      devServerProxy: this.props.devServerProxy || {},
+      maxChunkSize: this.props.maxChunkSize || 5000000,
+      workspaceAliases: this.props.workspaceAliases || [],
+      registry: this.props.registry || [],
+      vendors: this.props.vendors || []
```

Comment:
Could we move this to a separate file that is loaded and merged? Here it is quite hidden and difficult to find and debug.